config/cylc_hpc/flow.cylc

```diff
@@ -106,82 +102,89 @@
     
     [[pulljuliaimage]]
         script = """
-            apptainer pull --force $project_dir/icefloetracker-julia.simg docker://brownccv/icefloetracker-julia:pr-45
+            apptainer pull --force $project_dir/icefloetracker-julia.simg docker://brownccv/icefloetracker-julia:main
         """
         platform = oscar
         execution time limit = PT1H
         submission retry delays = 2*PT30S
         execution retry delays = 2*PT6S
         [[[directives]]]
-            --mem = 16G
+            --mem = 32G
             --cpus-per-task = 8
```

Comment:
I guess just one core is enough to pull the images from docker hub? Is this done only once?

Reply:
Good call, this is done once, but singularity will check each of the image blobs each time the task is run to make sure everything is current